src-vue/src/utils/ui.ts

```diff
@@ -1,15 +1,21 @@
 import { ElNotification, NotificationHandle } from "element-plus";
 import { i18n } from "../main";
+import { store } from "../plugins/store";
 
 /**
  * Displays content to the user in the form of a notification appearing on screen bottom right.
+ * If the app is not focused when this is invoked, a notification is added to the notifications menu.
  **/
 function showNotification(
     title: string,
     message: string = '',
     type: 'success' | 'warning' | 'error' | 'info' = 'success',
     duration: number = 4500
 ): NotificationHandle {
+    if (!document.hasFocus()) {
+        store.commit('addNotification', {title, text: message, type});
```

Comment:
I think there's probably also something in the Tauri API to make the taskbar icon flash, although I'm fine with moving that feature to a separate PR \^\^

Reply:
Moved to #634